### ISSUE TITLE
Explore: Get Started

### ISSUE THREAD

**Issue by meanmina** [grafana/grafana#61712]
Investigate potential ways of improving onboarding of new users.

```[tasklist]
### @radiorental conducted some research and we invested in:
- [x] https://github.com/grafana/grafana/issues/57652 (when no DS setup, guide user to create DS - measure creation through this new view)
- [x] Explore New User Journey mapping [HERE](https://www.figma.com/file/y8CwLz66bWeiLbvMSrxSn8/Explore-New-User-Journey?node-id=0%3A1&t=zo2j3i6CUlxYVXKi-0) and identify quick wins
- [x] #61945
```

```[tasklist]
### @diegoadams is currently conducting more research:
- [x] Create [research plan](https://docs.google.com/document/d/1xL1r9LMXhymZqhB1orjsg9wbkqs-9Xgtfqgs6inTPqg/edit#)
- [ ] `May 29, 2023`  -This research draft and testing script completed
- [ ] `Jul 7, 2023` Recruitment for external users completed
- [ ] `Apr 21, 2023` Interviews concluded
- [ ] `Apr 26, 2023` Results presented & submitted:[Research Findings: Getting Started Explore](https://docs.google.com/presentation/d/15crQmWYhRlwxBJx0hezvDyKsV_IwwUtKo-ZJksBrsnI/edit#slide=id.g200f1f3901f_0_252)
```

Other ideas:
- picking a datasource for first time users
- check with Alerting how their measured value of their how-to page
- implement a way to detect if first-time user
- do something clever with Grafana DS to guide the user (Kristina and Gio's idea)
     - Have more user-understandable and guided scenarios with a datasource that is most likely installed by default, to give a new user something to play with that doesn't require action on their part.
- Review onboarding data to see where drop-offs happen
- Helping user understand why their queries may not work (@Elfo404) - Connections Console Roadmap
Given users can save datasources in a non-working state (ie. by not entering a valid url) the idea was to save the last tested state in the DB and show some sort of warning when selected said datasource (maybe in the ds picker, maybe somewhere else). this should help users getting unblocked if the datasource they are trying to use is not correctly set up.

We need to discuss this possibly with user-essentials, dashboards, or other squads to check if this can benefit multiple areas and if it makes sense in the first place (https://raintank-corp.slack.com/archives/C034LMXLXQU/p1673954973182649)

**Comment by gelicia** [comment]
This is not at all a "quick win" but a stretch goal may be a how to page that's provisionable so cloud users can opt into having more test data set up with a how to page written to take advantage of that test data for guidance. I'm thinking a checkbox when you go to launch your new stack that would let them opt into a more guided experience that requires less setup on their side.

This would be in contrast with an out of the box OSS install that might have a more generic default.
